frontend/src/utils/documentListHelper.js

```diff
@@ -347,6 +347,44 @@ export function getRowsData(rowData) {
   return data;
 }
 
+/**
+ * @method formatStringWithZeroSplitBy
+ * @summary For a given string with a specified separator it formats the number within with zeros in front if they are below 10
+ * @param {string} date
+ * @param {string} notation
+ */
+export function formatStringWithZeroSplitBy(date, notation) {
```

Comment:
i think u don't have to export `formatStringWithZeroSplitBy`.
does not look like it's used outside of this file.

Reply:
Yep. removed. Thanks for pointing this out.